database/src/main/resources/changesets/metadata_journal.xml

```diff
@@ -0,0 +1,67 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.3.xsd">
+
+    <changeSet author="mcovarr" id="metadata_journal">
+        <createTable tableName="METADATA_JOURNAL">
+            <column autoIncrement="true" name="METADATA_JOURNAL_ID" type="INT">
+                <constraints primaryKey="true" primaryKeyName="PK_METADATA_JOURNAL"/>
+            </column>
+            <column name="WORKFLOW_EXECUTION_UUID" type="VARCHAR(255)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="METADATA_KEY" type="VARCHAR(255)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="METADATA_CALL_FQN" type="VARCHAR(255)">
+                <constraints nullable="true"/>
+            </column>
+            <column name="METADATA_CALL_INDEX" type="INT">
+                <constraints nullable="true"/>
+            </column>
+            <column name="METADATA_CALL_ATTEMPT" type="INT">
+                <constraints nullable="true"/>
+            </column>
+            <column name="METADATA_VALUE" type="TEXT">
+                <constraints nullable="true"/>
+            </column>
+            <column name="METADATA_TIMESTAMP" type="DATETIME">
+                <constraints nullable="false"/>
+            </column>
+        </createTable>
+
+        <modifySql dbms="mysql">
+            <append value=" ENGINE=INNODB"/>
+        </modifySql>
+    </changeSet>
+
+    <changeSet id="metadata_workflow_index" author="mcovarr">
+        <createIndex indexName="METADATA_WORKFLOW_IDX"
+                     tableName="METADATA_JOURNAL" unique="false">
+            <column name="WORKFLOW_EXECUTION_UUID"/>
+        </createIndex>
+    </changeSet>
+
+    <changeSet id="metadata_job_index" author="mcovarr">
+        <createIndex indexName="METADATA_JOB_IDX"
+                     tableName="METADATA_JOURNAL" unique="false">
+            <column name="WORKFLOW_EXECUTION_UUID"/>
+            <column name="METADATA_CALL_FQN"/>
+            <column name="METADATA_CALL_INDEX"/>
+            <column name="METADATA_CALL_ATTEMPT"/>
+        </createIndex>
+    </changeSet>
+
+    <changeSet id="metadata_key_index" author="mcovarr">
+        <createIndex indexName="METADATA_KEY_IDX"
+                     tableName="METADATA_JOURNAL" unique="false">
+            <column name="WORKFLOW_EXECUTION_UUID"/>
+            <column name="METADATA_KEY"/>
+            <column name="METADATA_CALL_FQN"/>
+            <column name="METADATA_CALL_INDEX"/>
+            <column name="METADATA_CALL_ATTEMPT"/>
+        </createIndex>
+    </changeSet>
```

Comment:
Spec assertions for indexes differences are turned [off](https://github.com/broadinstitute/cromwell/blob/0.19/engine/src/main/scala/cromwell/engine/db/DiffResultFilter.scala#L31-L51) for the moment, until I have more time to investigate some issues when comparing slick vs liquibase schemas.

Still, please add the defined index above to the slick schema in `MetadataComponent`. See the `WorkflowExecutionComponent` for an example: [liquibase](https://github.com/broadinstitute/cromwell/blob/0.19/engine/src/main/resources/changesets/db_schema.xml#L135-L138) & then [slick](https://github.com/broadinstitute/cromwell/blob/0.19/engine/src/main/scala/cromwell/engine/db/slick/WorkflowExecutionComponent.scala#L41)